bolt-design-system/bolt
# Code-snippet | Bug | Fix for code-snippet component to correct work without JS

## Jira

http://vjira2:8080/browse/BDS-1479

## Summary

Update twig version of code-snippet to be consistent with JS version.

## Details

(Explain the changes in enough detail fo reviewers to understand.  Raise any questions for reviewers to consider.)

## How to test

Run this code locally. Go to `code-snippet` page and test behavior by disabling and enabling of JS on specific subpages. Only differences in both versions should be syntax highlighting.
####TWIG version:
![image](https://user-images.githubusercontent.com/16049049/60029710-f53fa200-96a1-11e9-93a3-9c6f3d198e75.png)
![image](https://user-images.githubusercontent.com/16049049/60029743-fec90a00-96a1-11e9-856d-1613154ee2e8.png)
####JS version:
![image](https://user-images.githubusercontent.com/16049049/60029637-d214f280-96a1-11e9-903d-c7b757eadd27.png)
![image](https://user-images.githubusercontent.com/16049049/60029683-e6f18600-96a1-11e9-831e-ed3fe5077623.png)
